### PR TITLE
Create .travis.yml

### PR DESCRIPTION
**Pull request by reagere**

### CODE REVIEW

**Comment by reagere** [comment]
ça passe, mais sur la forme c'est pas top.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+go:
+  - 1.x
+
+cache:
+  yarn: true
+
+addons:
+  apt_packages:
+    - pandoc
```
Comment (jfbus) [inline .travis.yml]: ???

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+before_install:
+  - export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)"
+  - echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+  - curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+  - sudo apt-get update && sudo apt-get install -y google-cloud-sdk google-cloud-sdk-pubsub-emulator
+#  - go get github.com/mattn/goveralls
+
+before_script:
+  - screen -S emu -d -m gcloud beta emulators pubsub start
+  - sleep 5s
```
Comment (jfbus) [inline .travis.yml]: mandatory ?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  - $(gcloud beta emulators pubsub env-init)
+
+#script:
+#  - $GOPATH/bin/goveralls -service=travis-ci
```
Comment (jfbus) [inline .travis.yml]: don't forget to uncomment this...

File: pubsub/pubsub_test.go [hunk 4]
```diff
@@ -186,8 +186,13 @@ func TestServerWithMultipleEndpoints(t *testing.T) {
 	go func() {
 		exitError <- srv.Run(ctx)
 	}()
+	i := 0
 	for tr.c == nil {
 		time.Sleep(time.Millisecond)
+		if i > 10000 {
```
Comment (jfbus) [inline pubsub/pubsub_test.go]: looks like a ++ is missing somewhere

File: pubsub/pubsub_test.go [hunk 5]
```diff
@@ -186,8 +186,13 @@ func TestServerWithMultipleEndpoints(t *testing.T) {
...
+			t.Errorf("more than 10 seconds to get google pubsub client")
+			break
+		}
 	}
 
 	{
```
Comment (jfbus) [inline pubsub/pubsub_test.go]: There are still a lot of references to cloud tasks below this line...

File: pubsub/pubsub_test.go [hunk 6]
```diff
@@ -186,8 +186,13 @@ func TestServerWithMultipleEndpoints(t *testing.T) {
 	go func() {
 		exitError <- srv.Run(ctx)
 	}()
+	i := 0
 	for tr.c == nil {
```
Comment (jfbus) [inline pubsub/pubsub_test.go]: why use tr.c and not c ?

File: pubsub/pubsub.go [hunk 7]
```diff
@@ -45,12 +45,14 @@ func (t *Transport) Start(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	exit := make(chan error)
```
Comment (jfbus) [inline pubsub/pubsub.go]: this leaks goroutines and might miss an error.

```
var g errgroup.Group
for _, e := range t.endpoints {
	endpoint := e
	g.Go(func() error {
		return t.consume(ctx, endpoint)
	})
}
return g.Wait()
```

**Pull request merged**